app/gui2/src/providers/widgetRegistry/editHandler.ts

```diff
@@ -187,6 +205,7 @@ class PortEditInteraction implements Interaction {
   readonly active: Ref<boolean>
   private readonly resumable: PortEditResumeData
   private readonly interactions = new Array<PortEditSubinteraction>()
+  private child: PortEditInteraction | undefined = undefined
```

Comment:
The docs say that `PortEditInteraction` is a 
> Top-level interaction active when a port is being edited by associated widgets.

But here child is also `PortEditInteraction`, and clearly it cannot be top-level, as it is a child. Also, we already have `interactions` field - why do we need additional `child` field?